contracts/minter-suite/minter_set_price_v1.vy

```diff
@@ -0,0 +1,252 @@
+# @version ^0.3.7
+
+"""
+@title Filtered Minter contract that allows tokens to be minted with ETH.
```

Comment:
I think this is worth having in repo as a working prototype/reference implementation that has not been audited. It also only integrates with our V1 core, not v3 core (e.g. call to `onlyWhitelisted`).

From that standpoint, I think my only suggestions are to move this into some kind of reference/prototype directory, and highlighting the fact that this is a prototype and not audited in the contract name and/or comments 👍